test/spec/BuildSpec.hs

```diff
@@ -8,32 +8,13 @@ import Development.Shake (CmdOption (EchoStdout), StdoutTrim (..), cmd, cmd_)
 import System.Directory
 import System.Exit (ExitCode (..))
 import Test.Hspec
-import Test.Mockery.Directory
-import Test.Mockery.Environment (withModifiedEnvironment)
 import TestUtils
 
-wrap :: SpecWith ([String] -> IO ProcResult) -> Spec
-wrap spec = do
-  repoDir <- runIO getCurrentDirectory
-  around_
-    ( withModifiedEnvironment [("NIX_CONFIG", "experimental-features =")]
-        . inTempDirectory
-    )
-    $ around
-      ( \test -> onTestFailureLogger $ \onTestFailureLog -> do
-          let runGarn' args = do
-                output <- runGarn args "" repoDir Nothing
-                onTestFailureLog output
-                return output
-          test runGarn'
-      )
-    $ spec
-
 spec :: Spec
 spec = do
   describe "build" $ do
     repoDir <- runIO getCurrentDirectory
-    wrap $ do
+    withRunGarn $ do
```

Comment:
Maybe move this to immediately after `describe "build"` for consistency with the file below, or vice-versa? Definitely TIOLI, though

Reply:
Yup, done.